index.html

```diff
@@ -315,10 +315,10 @@ <h2 class="ab-title new">Skills</h2>
       <div class="contact-me">
         <div class="contact-sec">
           <div class="close">
-            <h1 class="closure">
+            <p class="closure">
               I'm always interested in hearing about new projects, so if you'd
               like to chat please get in touch.
-            </h1>
+            </p>
```

Comment:
[OPTIONAL] Great job making the requested changes :clap: , I think because you changed this to a `p` tag now the text is not bolad as it should be according to the figma design, you can do this by setting appropriate `font-weight` to this paragraph
your design :point_down: 
![image](https://user-images.githubusercontent.com/97521557/200136581-2538e29e-fd35-4f1a-8983-e1496f732b5f.png)
figma template
![image](https://user-images.githubusercontent.com/26736582/200122347-17b90079-9f5c-48de-ae82-9f9603bf4245.png)